src/components/console/RequestBlock/RequestBlock.js

```diff
@@ -0,0 +1,101 @@
+    const [responseValue,setResponseValue] = useState("")
+    const {addRequest, requestInputValue, setRequestInputValue, needSubmit, setNeedSubmit} = useConsoleContext()
+
+    const onSubmit = useCallback(
+         () => {
+            setIsServerError(false)
+            try {
+                const request = JSON.parse(requestInputValue)       
+                api.sendsay.request(request)
+                    .then((res)=> {
+                        addRequest({request:{...request}, response: {...res}, success: true})
+                        setResponseValue(JSON.stringify(res,null,'\t'))})
+                    .catch((err)=>{ 
+                        addRequest({request:{...request}, response: {...err}, success: false})
+                        setIsServerError(true)
+                        setResponseValue(JSON.stringify(err,null,'\t'))});
+            } catch (error) {
+                setResponseValue('')
+                setIsError(true)
+            }     
+        },
+        [requestInputValue,addRequest],
+    )
+
+    useEffect(() => {
+        if(needSubmit){
+            onSubmit()
+            setNeedSubmit(false)
+        }
+    }, [needSubmit,onSubmit,setNeedSubmit])
+
+    const onTextareaChange = (e) => {
+        isError && setIsError(false)
+        setRequestInputValue(e.target.value) 
+    }
+
+    const onFormatingClick = () => {
+        try {     
+            setRequestInputValue(JSON.stringify(JSON.parse(requestInputValue),null,'\t'))
+        } catch (error) {
+            setIsError(true)
+        } 
+    }
+    
+    return (      
+        <div className={styles.wrap} >
+            <div className={styles.root}>
+            <div className={styles.blockWrap} >
+                <span className={clsx(styles.span, isError && styles.error)}>Запрос:</span>
+                <div className={clsx(styles.reqresBlock, isError && styles.errorBlock) }>        
+                        <textarea   name="requestInput" 
+                                    id="requestInput" 
+                                    value={requestInputValue}
+                                    type="text" 
+                                    onChange={onTextareaChange}/>                    
+                </div>
+            </div>
+            <div className={styles.imgWrap}><img src={dots} alt="dots" /></div>
+            <div className={styles.blockWrap}>
+                <span className={clsx(styles.span, isServerError && styles.error)}>Ответ:</span>
+                <div className={clsx(styles.reqresBlock, isServerError && styles.errorBlock) }>
+                    <pre>
+                        {responseValue}
+                    </pre>
+                </div>
+            </div>
+            </div>
+            
+            <div className={styles.submitBlock}>
+                <PrimaryButton type="submit" onClick={onSubmit}>
+                { false ? <img alt='loading' src={spinner}/>:'Отправить'}
```

Comment:
Please write ternary operators in a right way. Each condition from a new line. 
Like you did here.
https://github.com/Sa6aTeur/API-console/pull/1/files#diff-05c96ccfe038b092ee29604adbbfa12d42f32fe5441fe760e3bc39227c60b3f4R51

Reply:
this comment is done

Reply:
LGTM